config.yml.template

```diff
@@ -5,6 +5,7 @@ email:
   imapUrl: imap.gmail.com
   smtpUrl: smtp.gmail.com
   smtpPort: 587
+  gmailUrl: https://mail.google.com/
```

Comment:
to be clearer, when I mentioned hardcoding the gmail addresses, I really just meant putting them in the code as constant strings -- it's not like this will work for anything besides gmail anyway

Reply:
we can just hardcode imapUrl, smtpUrl, smtpPort, and gmailUrl in the code itself rather than have them be in the config file

Reply:
created strings to hold those values